### PR TITLE
Fixed start to run multiple machines and updated docs

### PR DESCRIPTION
**Pull request by DharminB**
- Fixes [this](https://github.com/HBRS-MAAS/project/pull/111#discussion_r248756167)
- Updated docs to share the information related to the changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,24 @@ Just install gradle and run:
...
+- Find the ip of the server/host machine (for example `192.168.1.123`)
+- Find a port which can be used for JADE communication (for example `5555`)
+
+For server/host machine
+
+    gradle run --args="-isHost 192.168.1.123 -port 5555 -stage1 -stage2 ... -noTK"
+
+For client machines
+
+    gradle run --args="-host 192.168.1.123 -localPort 5555 -stage3 -noTK"
```
Comment (Sushant-Chavan) [inline README.md]: I believe the -localPort is to be specified for the host and the -port for the clients.

File: README.md [hunk 2]
```diff
@@ -15,6 +15,24 @@ Just install gradle and run:
...
+    
+For the client which executes gradle command later than every other machine
+
+    gradle run --args="-host 192.168.1.123 -localPort 5555 -stageN"
```
Comment (Sushant-Chavan) [inline README.md]: Also here

File: docs/start.md [hunk 3]
```diff
@@ -1,12 +1,10 @@
...
 
-## Starting
-
-Possible to give commandline arguments:
+## Starting Possible to give commandline arguments: 
 
 |command            |effect                                             |
 | ------------------|:-------------------------------------------------:|
-|-isHost            |starts a main Container                            |
+|-isHost {port}     |starts a main Container                            |
```
Comment (Sushant-Chavan) [inline docs/start.md]: I think this should be ip address instead of port

File: README.md (hunk 1 above)
Comment (DharminB) [reply README.md]: Are you sure? I took a look at my bash history from tuesday evening and found it this way? I also tried running server and client on different terminals with the proposed commands. Could you check with multiple machines and let me know?

File: docs/start.md (hunk 3 above)
Comment (DharminB) [reply docs/start.md]: Fixed.

File: README.md (hunk 2 above)
Comment (DharminB) [reply README.md]: Could you test it with multiple machines?

File: README.md (hunk 1 above)
Comment (DharminB) [reply README.md]: I noticed that as well. Let me know if you have results from execution.

Comment (Sushant-Chavan) [reply README.md]: Hi @DharminB , 
We tested the changes with @HBRS-MAAS/jsw and I can confirm that the -localPort is to be specified for the host and the -port for the clients. Please make the change accordingly.

**Comment by Sushant-Chavan** [comment]
@DharminB can you merge these fixes on the develop branch?

**Comment by DharminB** [comment]
> @DharminB can you merge these fixes on the develop branch?

Done

**Pull request merged**